buildSrc/src/main/kotlin/io/embrace/gradle/Versions.kt

```diff
@@ -39,7 +39,10 @@ object Versions {
     val ndk = "21.4.7075529"
 
     @JvmField
-    val openTelemetry = "1.35.0"
+    val openTelemetryCore = "1.35.0"
+
+    @JvmField
+    val openTelementrySemConv = "1.24.0-alpha"
```

Comment:
Is there a stable channel that we can rely on instead?

Reply:
This is technically considered a stable release because some of the sem conv are considered experimental: https://github.com/open-telemetry/semantic-conventions-java/releases/tag/v1.24.0